src/components/DatePicker/DatePicker.st.css

```diff
@@ -203,10 +204,18 @@
 
 .calendar::day {
     font: value(CurrentDayFont);
-    font-size: 14px;
     color: value(CurrentDayColor);
-    width: 34px;
-    height: 34px;
+    width: calc(34em/14);
+    height: calc(34em/14);
+}
+
+.calendar::dayText {
+    line-height: 1;
```

Comment:
Maybe this should also be relative to `em` like the others?

Reply:
Actually, I also added this `line-height: 1;` for the WSR component. I added it here as well in order to verify that no one is overriding it.
It is required in order to align the dates that have an indication for those who don't.